reference/admin.yaml

```diff
@@ -64,6 +72,13 @@ components:
           description: >
             This error means the requested resource was not found or the
             requester doesn't have sufficient access.
+    '500':
+      type: object
+      properties:
+        message:
+          type: string
+          description: |
+            API server got an error and could not process request.
```

Comment:
API server hit an internal error and could not process request. This is likely a bug.